waiverdb/templates/permissions.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
Can you use "layout.html" here too as in "new_waiver.html" and move the non-common code to custom blocks (e.g. the `<script>`)?